daily-challenges/2024-09/16/hunghoang3110.spec.ts

```
@@ -0,0 +1,67 @@
+// Javascript
+function revertString(input) {
+    let rev_string = input.split("").reverse().join("");
+    console.log(`OUTPUT:\n${rev_string}`);
+}
+
+// Playwright
+import { test, expect } from '@playwright/test';
+import { randomInt } from 'crypto';
```

Biome (1.9.4)

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)